packages/ecommerce-app-base/src/SkuPicker/SkuPicker.tsx

```diff
@@ -148,35 +155,59 @@ export class SkuPicker extends Component<Props, State> {
   };
 
   render() {
-    const { search, pagination, products, selectedProducts, selectedSKUs } = this.state;
-    const { makeSaveBtnText = defaultGetSaveBtnText, skuType, hideSearch = false } = this.props;
+    const { search, pagination, products, selectedProducts, selectedSKUs, searchBySku } =
+      this.state;
+    const {
+      makeSaveBtnText = defaultGetSaveBtnText,
+      skuType,
+      hideSearch = false,
+      showSearchBySkuOption,
+    } = this.props;
     const infiniteScrollingPaginationMode = 'hasNextPage' in pagination;
     const pageCount = Math.ceil(pagination.total / pagination.limit);
 
     return (
       <>
         <header className={styles.header}>
-          <div className={styles.leftSideControls}>
+          <div>
             {!hideSearch && (
-              <>
-                <TextInput
-                  placeholder="Search for a product..."
-                  type="search"
-                  name="sku-search"
-                  id="sku-search"
-                  testId="sku-search"
-                  value={search}
-                  onChange={(event) => this.setSearch((event.target as HTMLInputElement).value)}
-                />
-                <SearchIcon variant="muted" />
-              </>
+              <div className={styles.searchWrapper}>
+                <div className={styles.leftSideControls}>
+                  <TextInput
+                    placeholder="Search for a product..."
+                    type="search"
+                    name="sku-search"
+                    id="sku-search"
+                    testId="sku-search"
+                    value={search}
+                    onChange={(event) => this.setSearch((event.target as HTMLInputElement).value)}
+                  />
+
+                  <SearchIcon variant="muted" />
+                </div>
+
+                {showSearchBySkuOption && (
+                  <div className={styles.skuSearch}>
+                    <Checkbox
+                      name="search-by-sku"
+                      testId="search-by-sku"
+                      id="search-by-sku"
+                      isChecked={searchBySku}
+                      onChange={() => this.setSearchBySku()}>
+                      Search only by SKU
+                    </Checkbox>
+                  </div>
+                )}
+              </div>
             )}
+
             {!!pagination.total && (
-              <span className={styles.total}>
-                Total results: {pagination.total.toLocaleString()}
-              </span>
+              <Text className={styles.total}>
+                {pagination.total.toLocaleString()} total results
+              </Text>
```

Comment:
Adjusted this here a per Emily's suggestion. Let me know if I misinterpreted this in any way!

Reply:
Confirmed with Emily that this is what she intended 💪